contrib/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/commands/WildflyManagementAPI.java

```diff
@@ -119,7 +119,7 @@ static Object createOperation(WildflyDeploymentFactory.WildFlyClassLoader cl, Ob
     // ModelNode
     static Object createReadResourceOperation(WildflyDeploymentFactory.WildFlyClassLoader cl, Object modelNode, boolean recursive)
             throws ClassNotFoundException, NoSuchMethodException, IllegalAccessException, InvocationTargetException {
-        Class clazz = cl.loadClass("org.jboss.as.controller.client.helpers.Operations"); // NOI18N
+        Class<?> clazz = cl.loadClass("org.jboss.as.controller.client.helpers.Operations"); // NOI18N
         Class modelClazz = cl.loadClass("org.jboss.dmr.ModelNode"); // NOI18N
```

Comment:
This looks strange - why was line 122 changed, but 123 not? This seems to be the case at multiple places.

Reply:
This is a result of the egrep cmd only picking up instances of "Class clazz".. This seemed like the easiest thing to do to reduce the warning message and do no harm at the same time.

Hopefully, I can reduce more of these messages and make the code review easy by doing more of this.. I really don't like creating a big code review because it makes more work for you and others.